components/atoms/Button/Button.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+ 
+import styled from "styled-components";
+
+const ButtonStyle = styled.button`
+    background:#bf2957;
+    font-family:'Myriad Pro Regular';
+    color:#fff;
+    border:none;
+    text-align: center;
+    width:auto;
+    height:30px;
+`; 
+
+const Button= (props) => {
+  const { children,type, cname } = props;
+  return (
+        <ButtonStyle className={cname} type={type}>{children}</ButtonStyle>
```

Comment:
IF more than one variant of buttons are required then pass the variant props and override the styling in button component only. Otherwise if specific css property needs to override then please override using styled component.